### PR TITLE
[ADD] pos_partner_ref module

### PR DESCRIPTION
**Pull request by cvinh**
(Inspired from pos_partner_birthdate improved by @etobella https://github.com/OCA/pos/pull/1118)
Adds the partner ref in the customer screen of POS

### CODE REVIEW

File: pos_partner_ref/static/src/js/ClientDetailsEdit.esm.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import {PartnerDetailsEdit} from "@point_of_sale/app/screens/partner_list/partner_editor/partner_editor";
+import {patch} from "@web/core/utils/patch";
+
+patch(PartnerDetailsEdit.prototype, {
+    setup() {
+        super.setup(...arguments);
+        this.changes.ref = this.props.partner.ref || null;
+    },
+    _today() {
+        return new Date().toISOString().split("T")[0];
```
Comment (chienandalu) [inline pos_partner_ref/static/src/js/ClientDetailsEdit.esm.js]: Is this needed at all or a remanent of copied code :sweat_smile: ?

Comment (cvinh) [reply pos_partner_ref/static/src/js/ClientDetailsEdit.esm.js]: thanks

**Review by chienandalu — APPROVED** [review]
Functional and code ok :+1:

**Review by etobella — APPROVED** [review]
:+1:  Technical and functional review

**Comment by cvinh** [comment]
I did a force push because there was a pandoc deb package in my first commit... BTW I don't know how it landed here

**Comment by cvinh** [comment]
@etobella my force push was done after the merge command, how to manage that ?

**Pull request merged**